src/runtime_src/doc/toc/xrt_native_apis.rst

```diff
@@ -0,0 +1,618 @@
+.. _xrt_native_apis.rst:
+
+XRT Native APIs
+===============
+
+From 2020.2 release XRT provides a new XRT API set in C, C++, and Python flavor. This document introduces the usability of C and C++ APIs.
+
+To use the native XRT APIs, the host application must link with the **xrt_coreutil** library.
+
+.. code-block:: shell
+
+    g++ -g -std=c++14 -I$XILINX_XRT/include -L$XILINX_XRT/lib -o host.exe host.cpp -lxrt_coreutil -pthread
+
+
+The core data structures in C and C++ are as below
+
++---------------+---------------+-------------------+
+|               |   C++ Class   |  C Type (Handle)  |
++===============+===============+===================+
+|   Device      | xrt::device   |  xrtDeviceHandle  |
++---------------+---------------+-------------------+
+|   XCLBIN      | xrt::xclbin   |  xrtXCLBINHandle  |
++---------------+---------------+-------------------+
+|   Buffer      | xrt::buffer   |  xrtBufferHandle  |
```

Comment:
xrt::bo (for buffer object) in hindsight maybe buffer would have been better.  If that is what we prefer, then I can create an alias.   The C type handle is indeed xrtBufferHandle

Reply:
Thanks, fixed.. done.